packages/browser-wallet/src/popup/pages/Account/SendCcd/SendCcd.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+import { Route, Routes, Navigate } from 'react-router-dom';
+import { SimpleTransferPayload } from '@concordium/web-sdk';
+import { getSimpleTransferCost } from '@popup/shared/utils/wallet-proxy';
+import { noOp, useAsyncMemo } from 'wallet-common-helpers';
+import CreateTransfer from './CreateTransfer';
+import ConfirmTransfer from './ConfirmTransfer';
+import { routes } from './routes';
+
+interface Props {
+    setDetailsExpanded: (expanded: boolean) => void;
+}
+
+export default function SendCcd({ setDetailsExpanded }: Props) {
+    const [payload, setPayload] = useState<SimpleTransferPayload>();
+    const cost = useAsyncMemo(getSimpleTransferCost, noOp, []);
```

Comment:
I don't get an error message if the cost could not be fetched, it is just shown as `0`. I guess it doesn't make sense to allow sending a transaction if we don't have the cost available when building the transaction?

Reply:
:thinking:
Yeah I guess we might want to disable continuing or something.